platforms/java/getJavaSdk.sh

```diff
@@ -46,6 +47,20 @@ downloadJavaSdk()
     IFS='.' read -ra VERSION_PARTS <<< "$JDK_VERSION"
     majorVersion="${VERSION_PARTS[0]}"
 
+    # Version 17 has a different url format than rest of the versions, so special casing it.
+    if [ "$majorVersion" == "17" ]; then
+        local buildNumber="12"
+        local url="https://download.java.net/java/GA/jdk${JDK_VERSION}/2a2082e5a09d4267845be086888add4f/${buildNumber}/GPL/openjdk-${JDK_VERSION}_linux-x64_bin.tar.gz"
+
+        curl -L "$url" -o $tarFileName
+        rm -rf extracted
+        mkdir -p extracted
+        tar -xf $tarFileName --directory extracted
+        tar -zcf "$hostJavaArtifactsDir/$tarFileName" .
+		echo "Version=$JDK_VERSION" >> "$hostJavaArtifactsDir/java-$JDK_VERSION-metadata.txt"
+        return
+    fi
+
     if shouldBuildSdk java $tarFileName || shouldOverwriteSdk || shouldOverwriteJavaSdk; then
         local baseUrl="https://github.com/AdoptOpenJDK/openjdk${majorVersion}-binaries/releases/download"
         if [ "$majorVersion" == "10" ] && [ "$JDK_BUILD_NUMBER" == "13" ]; then
```

Comment:
It seems JDK 17 is available in its own repository within this `AdoptOpenJDK` organization, so even though the URL is different for JDK 17, is there any reason we can't add an additional if-statement like we do here for 10.x.x.13 to special case the URL?

Reply:
`17.0.1` currently isn't supported: https://github.com/AdoptOpenJDK/openjdk17-binaries/releases/
Last release was in May 2021 & 17.0.1 was last released in September 2021

Reply:
look at dotnet core example for supporting multiple download link for sdks ... I'm approving this for one time to unblock you for releasing for vso

Reply:
here is an example https://github.com/microsoft/Oryx/blob/9c093d1d9780093a4c907c4222d2888c8bbe2304/platforms/dotnet/versionsToBuild.txt#L65